samples/csharp_dotnetcore/language-generation/05.a.multi-turn-prompt-with-language-fallback/AdapterWithErrorHandler.cs

```diff
@@ -8,12 +8,13 @@
 using Microsoft.Bot.Connector.Authentication;
 using Microsoft.Bot.Builder.Integration.AspNet.Core;
 using System.Collections.Generic;
+using Microsoft.Bot.Builder.LanguageGeneration;
 
 namespace Microsoft.BotBuilderSamples
 {
     public class AdapterWithErrorHandler : BotFrameworkHttpAdapter
     {
-        private MultiLingualTemplateEngine _lgManager;
+        private MultiLanguageLG _lgManager;
```

Comment:
>MultiLanguageLG [](start = 16, length = 15)

what the Wha?  How did this name ship?  We don't use acronyms in our class names!

Reply:
:( we could deprecate this in the future and fix up. Will file a tracking ticket.